jumpscale/packages/codeserver/package.toml

```diff
@@ -9,3 +9,4 @@ port = 8080
 path_url = "/codeserver"
 path_dest = "/"
 websocket = true
+increase_buffers_size = true
```

Comment:
I think we can just add the full options, not increase buffers size.
This won't fit all cases.